src/coercions.md

```diff
@@ -1,58 +1,16 @@
 # Coercions
 
-Types can implicitly be coerced to change in certain contexts. These changes are
-generally just *weakening* of types, largely focused around pointers and
-lifetimes. They mostly exist to make Rust "just work" in more cases, and are
-largely harmless.
+Types can implicitly be coerced to change in certain contexts.
+These changes are generally just *weakening* of types, largely focused around pointers and lifetimes.
+They mostly exist to make Rust "just work" in more cases, and are largely harmless.
 
-Here's all the kinds of coercion:
+For an exhaustive list of all the types of coercions, see the [Coercion types] section on the reference.
 
-Coercion is allowed between the following types:
+## Cases where coercions do not perform
```

Comment:
```suggestion
## Cases where coercions do not happen
```

(but I would drop the subheading altogether -- this is already a very short page now)